BASE/Test/Microsoft.ApplicationInsights.Test/Standalone/AppInsightsStandaloneTests.cs

```diff
@@ -30,6 +30,9 @@ public void Cleanup()
             Directory.Delete(this.tempPath, true);
         }
 
+#if NETCOREAPP2_1 || NETCOREAPP3_1
+        [Ignore("Missing netstandard dependencies.")]
```

Comment:
why ignored?

Reply:
There is a string code sample that's failing to compile. Seems like some dependencies are missing.

This is the `Isolated.Tests` project. It was never compiling in NetCore before now. This is the only test that is incompatible.
If you think this test needs to be run in NetCore builds I can spend some more time investigating.

Reply:
Full error:

> (12,32): error CS0012: The type 'IList<>' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'.
(14,58): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'.
(14,32): error CS0012: The type 'IDisposable' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'.
(14,32): error CS1674: 'IOperationHolder<RequestTelemetry>': type used in a using statement must be implicitly convertible to 'System.IDisposable'
(16,65): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'.
(16,36): error CS0012: The type 'IDisposable' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'.
(16,36): error CS1674: 'IOperationHolder<DependencyTelemetry>': type used in a using statement must be implicitly convertible to 'System.IDisposable'
(18,33): error CS0103: The name 'Console' does not exist in the current context
(18,67): error CS0012: The type 'IDisposable' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'.
(19,36): error CS0012: The type 'Object' is defined in an assembly that is not referenced. You must add a reference to assembly 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'.